src/app/lib/utilities/getUrlHelpers/getEmbedUrl/index.js

```diff
@@ -51,6 +62,14 @@ export default ({ type, mediaId, isAmp = false, queryString }) => {
   return `${url}${ampSection}${morphEnvOverride}`;
 };
 
+export default ({ type, mediaId, isAmp = false, queryString }) => {
+  const morphAsset = type !== 'avEmbed';
+  const embedUrl = morphAsset
+    ? handleMorphEmbed({ type, mediaId, isAmp, queryString })
+    : handleAvEmbed({ mediaPathname: mediaId });
```

Comment:
Is it fair to call the legacy function `handleMorphEmbed`?

Reply:
Yea I think its fine. We'll be removing/refactoring this in the next few weeks anyway when doing the AMP work, so references to Morph will be removed soon.